### DISCUSSION TITLE
Issues getting to OOBE stage

### DISCUSSION THREAD

**Discussion by lqbert** [memstechtips/UnattendedWinstall#64] [Q&A]
I'm attempting to install Win11 Pro as a VM with the unedited autounattend.xml file included in the ISO (I tried both Virtual Box and VMWare Player).  It never gets to the OOBE portion after multiple hours.  I get usually a black screen with a Windows "locked" type screen with the time showing, from time to time.  If I interact with the "lock" screen, it usually flashes back to the black screen.  Sometimes the lock screen stays visible long enough for me to press "Enter" and see a login screen for "Default User", but then it immediately goes back to the black screen or the lock screen.... very flaky acting... any ideas on how to go about fixing this?

**Comment by memstechtips** [comment]
Hey there and sorry for the late reply,

Since the creation of this issue I have released a new version of the XML file and a major update to [Winhance](https://winhance.net/), I recommend that you watch [this video](https://youtu.be/lrq3ph3xi50) which provides a full demo of the current version along with how to use the new version of WIMUtil and how to install the drivers etc. You now also have the ability to create an autounattend.xml file based on your selections in Winhance, so you have more control over which apps are removed and which settings are applied.

I have also created [this video](https://youtu.be/sCd1TI6AbOI) about v3.0.0 of UnattendedWinstall, I suggest that you watch it too if you're interested.

With v3.0.0 of the file, you shouldn't have this issue anymore.

I'm closing this issue now, but if you need more help with the same issue, feel free to reopen it.
